modyn/supervisor/internal/triggers/drift/detector/evidently.py

```diff
@@ -141,3 +150,25 @@ def _evidently_metric_factory(config: EvidentlyDriftMetric) -> EmbeddingsDriftMe
         )
 
     raise NotImplementedError(f"Metric {config.id} is not supported in EvidentlyDriftMetric.")
+
+
+def _evidently_additional_metric_computation(
+    configs: dict[str, EvidentlyDriftMetric],
+    embeddings_ref: pd.DataFrame,
+    embeddings_cur: pd.DataFrame,
+) -> dict[str, MetricResult]:
+    metric_results: dict[str, MetricResult] = {}
+    for metric_ref, config in configs.items():
+        if config.id == "EvidentlyHellingerDistanceDriftMetric":
+            column_distances = [
+                # [0]: Hellinger distance, [1]: decision with dummy threshold (False)
+                _hellinger_distance(embeddings_ref[c], embeddings_cur[c], ColumnType.Numerical, 0)[0]
+                for c in embeddings_ref.columns
```

Comment:
Each column is a data point, and the rows in this column vector are the embedding dimensions? (in my head it was the other way around just want to make sure we supply it the right way)